reactnative/reactnative-todo/android/app/src/main/java/com/reactnativetodo/Configuration.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2024 Ping Identity. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package com.reactnativetodo
+
+
+/**
+ * Configuration class for the ForgeRock SDK.
+ */
+data class Configuration(
+    /** The main authentication journey name.  */
+    val mainAuthenticationJourney: String = "jey-webAuthn",
```

Comment:
Can we replace this config with placeholder? We should expose details of our environments publicly

Reply:
I agree replace all values with [Placeholders]

Reply:
ha i accidentally committed that ..good catch . Fixed now